programs/marginfi/src/state/marginfi_group.rs

```diff
@@ -1067,12 +1118,14 @@ pub struct BankConfig {
     /// This is useful for limiting the damage of orcale attacks.
     ///
     /// Value is UI USD value, for example value 100 -> $100
-    pub total_asset_value_init_limit: u64,
+    pub total_asset_value_init_limit: u64, // 784 - 792
 
     /// Time window in seconds for the oracle price feed to be considered live.
-    pub oracle_max_age: u16,
+    pub oracle_max_age: u16, // 792  - 794
+
+    pub _pad1: [u8; 6], // 1x u16 + 6 = 8
```

Comment:
1. Wouldn't this result in a termination padding 2 bytes shorter? (`19 * 2 = 38` vs `6 + 30 = 36`)
2. Why the separation between `_pad1` and `_padding` instead of a single u8 padding array?

Reply:
1) You are correct it's 32 (doesn't actually matter it adds any missing bytes up to the alignment)
2) Either way is fine, I like to start the final padding at the correct alignment so nobody forgets to align if they add a variable later.

Reply:
1. Just for my OCD sake can you make it 32?
3. 👍